config.yml

```diff
@@ -69,5 +72,22 @@ presets:
+    inferno_uri: http://localhost:4567
+    bulk_url: https://bulk-data.smarthealthit.org/eyJlcnIiOiIiLCJwYWdlIjoxMDAwLCJkdXIiOjEwLCJ0bHQiOjE1LCJtIjoxLCJzdHUiOjR9/fhir
+    bulk_token_endpoint: https://bulk-data.smarthealthit.org/auth/token
+    bulk_requires_auth: true
+    bulk_client_id: eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCIsImtpZCI6InJlZ2lzdHJhdGlvbi10b2tlbiJ9.eyJqd2tzIjp7ImtleXMiOlt7Imt0eSI6IkVDIiwiY3J2IjoiUC0zODQiLCJ4IjoidFZ6RFdRNXFNMWVrUmtPOWFjVjJPckRXX0p5ZVM2YmFEclhQczl2UE9HT3hhNXo1NUNXdHB3Z19CRHlPWXQ4biIsInkiOiJocFhJQ1pNTmV6RDFvTU5ha09xbEdOaDNXUWdDZEtmUVVwRE5saHhfZEtRbzB5bU1QZVhLeGlzTW1WcVdPay1MIiwia2V5X29wcyI6WyJ2ZXJpZnkiXSwiZXh0Ijp0cnVlLCJraWQiOiJjOTUyOWEwYWFlZmE1ZWE2NTlkNjVjNDkzOWVjYTk3NiIsImFsZyI6IkVTMzg0In0seyJrdHkiOiJFQyIsImNydiI6IlAtMzg0IiwiZCI6Im05Z0pxR2FoOEM4LXZtb3M1LTZicDkwMlRQRkdIdmN0VUl6R2QyMkxQeUU2bzVEMldFVTFtV0VtNzRsRWV5YlAiLCJ4IjoidFZ6RFdRNXFNMWVrUmtPOWFjVjJPckRXX0p5ZVM2YmFEclhQczl2UE9HT3hhNXo1NUNXdHB3Z19CRHlPWXQ4biIsInkiOiJocFhJQ1pNTmV6RDFvTU5ha09xbEdOaDNXUWdDZEtmUVVwRE5saHhfZEtRbzB5bU1QZVhLeGlzTW1WcVdPay1MIiwia2V5X29wcyI6WyJzaWduIl0sImV4dCI6dHJ1ZSwia2lkIjoiYzk1MjlhMGFhZWZhNWVhNjU5ZDY1YzQ5MzllY2E5NzYiLCJhbGciOiJFUzM4NCJ9XX0sImFjY2Vzc1Rva2Vuc0V4cGlyZUluIjoxNSwiaWF0IjoxNTc0MDQ0MjIyfQ.dkqFAJreUXIfBTESdygmiOBhtepzgcDNJWqI2662m4Q
+    bulk_public_key: "{'kty':'EC','crv':'P-384','x':'tVzDWQ5qM1ekRkO9acV2OrDW_JyeS6baDrXPs9vPOGOxa5z55CWtpwg_BDyOYt8n','y':'hpXICZMNezD1oMNakOqlGNh3WQgCdKfQUpDNlhx_dKQo0ymMPeXKxisMmVqWOk-L','key_ops':['verify'],'ext':true,'kid':'c9529a0aaefa5ea659d65c4939eca976','alg':'ES384'}"
+    bulk_private_key: "{'kty':'EC','crv':'P-384','d':'m9gJqGah8C8-vmos5-6bp902TPFGHvctUIzGd22LPyE6o5D2WEU1mWEm74lEeybP','x':'tVzDWQ5qM1ekRkO9acV2OrDW_JyeS6baDrXPs9vPOGOxa5z55CWtpwg_BDyOYt8n','y':'hpXICZMNezD1oMNakOqlGNh3WQgCdKfQUpDNlhx_dKQo0ymMPeXKxisMmVqWOk-L','key_ops':['sign'],'ext':true,'kid':'c9529a0aaefa5ea659d65c4939eca976','alg':'ES384'}"
+    bulk_fastest_resource: Patient
+    bulk_system_export_endpoint: /$export
+    bulk_patient_export_endpoint: /Patient/$export
+    bulk_group_export_endpoint: /Group/5/$export
+    #https://bulk-data.smarthealthit.org/?auth_type=jwks&jwks=eyJrZXlzIjpbeyJrdHkiOiJFQyIsImNydiI6IlAtMzg0IiwieCI6InRWekRXUTVxTTFla1JrTzlhY1YyT3JEV19KeWVTNmJhRHJYUHM5dlBPR094YTV6NTVDV3Rwd2dfQkR5T1l0OG4iLCJ5IjoiaHBYSUNaTU5lekQxb01OYWtPcWxHTmgzV1FnQ2RLZlFVcERObGh4X2RLUW8weW1NUGVYS3hpc01tVnFXT2stTCIsImtleV9vcHMiOlsidmVyaWZ5Il0sImV4dCI6dHJ1ZSwia2lkIjoiYzk1MjlhMGFhZWZhNWVhNjU5ZDY1YzQ5MzllY2E5NzYiLCJhbGciOiJFUzM4NCJ9LHsia3R5IjoiRUMiLCJjcnYiOiJQLTM4NCIsImQiOiJtOWdKcUdhaDhDOC12bW9zNS02YnA5MDJUUEZHSHZjdFVJekdkMjJMUHlFNm81RDJXRVUxbVdFbTc0bEVleWJQIiwieCI6InRWekRXUTVxTTFla1JrTzlhY1YyT3JEV19KeWVTNmJhRHJYUHM5dlBPR094YTV6NTVDV3Rwd2dfQkR5T1l0OG4iLCJ5IjoiaHBYSUNaTU5lekQxb01OYWtPcWxHTmgzV1FnQ2RLZlFVcERObGh4X2RLUW8weW1NUGVYS3hpc01tVnFXT2stTCIsImtleV9vcHMiOlsic2lnbiJdLCJleHQiOnRydWUsImtpZCI6ImM5NTI5YTBhYWVmYTVlYTY1OWQ2NWM0OTM5ZWNhOTc2IiwiYWxnIjoiRVMzODQifV19&page=1000&stu=4
```

Comment:
Do we need this comment?

Reply:
This link allows you to view the configuration in their UI, which is useful if you want to tweak it at all.  I added a bit more of an explanation.